package/gluon-status-page/javascript/status-page.js

```diff
@@ -206,7 +206,7 @@
 			tr.appendChild(th);
 
 			var td = document.createElement('td');
-			if (peer[1])
+			if (peer[1] && Object.keys(peer[1]).length !== 0)
```

Comment:
Why not check for the established field, as that is what we're interested in?
```suggestion
			if (peer[1] && peer[1].established != null)
```
(this will actually be `undefined` when the field is not set, however `!= null` seems to be a common idiom to check for both `null` and `undefined`)

Reply:
Because I found the undefined part looked uglier. But I've got no strong opinions regarding any js code. If you want that instead, I'll update the PR tomorrow or so.

Reply:
What I feared was the undefined coming up in the developer console, like it does for updateNodeinfo and the location part. (another issue I'd like to get to in the next days...)

Reply:
resolved in b5242a530e0893e591f0f09cadefac5dc93cdf17